### ISSUE TITLE
svn training mission doesn't work

### ISSUE THREAD

**Issue by jwm** [openhatch/oh-mainline#1709]
When I deployed the production site to Heroku and went through a svn training mission, I noticed that http://openhatch.org/missions/svn/checkout wasn't displaying the URL to check out the user's repository, or the filename containing the secret word:

![screen shot 2015-09-29 at 8 14 12 pm](https://cloud.githubusercontent.com/assets/37176/10181478/af174506-66e6-11e5-8039-28973a10db26.png)

This is the first I've seen this, and I can't reproduce it by running the site locally. I don't *think* this is related to the Heroku deployment, since these are variables that are simply passed into the template for display; the site doesn't need to contact the remote repository server at all.

Additionally, the svn training mission fails further along the line because I overlooked the non-reset/create operations the view helper implements. They're being run on the Heroku dyno, which doesn't have the repositories on it. Those parts of code (in `mysite/missions/svn/view_helpers.py`) could be modified to use the remote repository access method (SSHing into linode.openhatch.org) like the parts of the code that creates and resets the repos. However, it might be better to apply a secret password to all svn repositories and have that code run the 'svn' command with the URL to the remote repository as an argument, which would probably be faster than SSHing into the remote server for each command.

For that matter, maybe it's time to remove the svn mission in favor of the git mission? git experience seems much more relevant now, and svn much less relevant than when we first added that mission.

**Comment by willingc** [comment]
@jwm Yup the SVN mission has some known issues. I think that it may be time to scale back time spent on that mission and simplify it more like the shell or pip missions. I am +1 to refactoring the svn missions so that we don't have to configure multiple servers and put less emphasis on it.

**Comment by ehashman** [comment]
+1 to a refactor of this mission, which has been super problematic for nitpicky bugs in my experience.

**Comment by Pythonek** [comment]
I am stuck on this mission http://openhatch.org/missions/svn/checkout
There is no url to check out repository.
Did someone resolve this problem?
